scripts/auto-rebase/rebase.sh

```diff
@@ -40,6 +40,8 @@ cluster-csi-snapshot-controller-operator"
 declare -a ARCHS=("amd64" "arm64")
 declare -A GOARCH_TO_UNAME_MAP=( ["amd64"]="x86_64" ["arm64"]="aarch64" )
 
+LVMS_RELEASE_MULTI_ARCH="registry.access.redhat.com/lvms4/lvms-operator-bundle:v4.13.1-5"
```

Comment:
Should we allow environment override here?

Reply:
The point of this change is to *not* allow anything outside of the script to influence the source of LVMS. I'll provide a better fix as part of USHIFT-1521, but we need something quick so we can get an EC build this sprint.